playbooks/openedx_native.yml

```diff
@@ -32,11 +32,12 @@
     SANDBOX_ENABLE_INSIGHTS: true
     SANDBOX_ENABLE_RABBITMQ: true
     JOURNALS_ENABLED: false
+    EDX_NOTES_API_ENABLED: false
   roles:
     - role: swapfile
       SWAPFILE_SIZE: 4GB
     - role: aws
-      when: COMMON_ENABLE_AWS_ROLE
+      when: COMMON_ENABLE_AWS_ROLE or COMMON_ENABLE_AWS_INTEGRATION
```

Comment:
If you're going to add this here, then best to also add a [default of `false` above](https://github.com/open-craft/configuration/pull/95/files#diff-ae99e0327f2aceaa880f7b146a55b8faR26).

Reply:
@pomegranited, thanks for catching this! This will eventually get removed because of the changes in the corresponding Ocim PR https://github.com/open-craft/opencraft/pull/430/files